Issues 290 & 291 | Fix header

vladplesu opened the pull request:
Fixes #290, #291 

Changed the header nodes to make use of the antd `Menu` components default behavior. Now for smaller devices the overflowed `Menu.Items` are shown in a sub-menu. This also fixes the problem with the menu overlapping with the description.

![header](https://i.gyazo.com/80eae10629fb2b487b4831a6464cb85f.gif)

tudoramariei commented:
I only have a couple of things to say about this:

1. I'd like if it had two states, one in which all the menu appears and another in which the whole menu is hidden under that button.
2. I'm not sure about that button (mostly because I'm accustomed to the three horizontal lines) but if it's something easy to change, I think it's going to be a UI question

The PR generally looks good and everything is hidden nicely. Thanks, Vlad!

vladplesu commented:
> 
> 
> I only have a couple of things to say about this:
> 
>     1. I'd like if it had two states, one in which all the menu appears and another in which the whole menu is hidden under that button.
> 
>     2. I'm not sure about that button (mostly because I'm accustomed to the three horizontal lines) but if it's something easy to change, I think it's going to be a UI question
> 
> 
> The PR generally looks good and everything is hidden nicely. Thanks, Vlad!

I`ll use the breakpoints from the blog page and make all the menu buttons collapse for mobile and tablets. Here is what I have in mind. 

![image](https://user-images.githubusercontent.com/22684808/88457868-7283aa80-ce92-11ea-829c-d9ce55b0dfe9.png)

Which expanded version do you prefer: expanded and move content or expanded with overlay ?

https://www.figma.com/file/GEJYaSPWvRYO9ouFWB59ft/Seismic-Risk-Menu

tudoramariei commented:
I believe the overlay looks best. If you can manage that one, it's perfect ✨

vladplesu commented:
Hello,

Modified the header as discussed above.

[![Image from Gyazo](https://i.gyazo.com/55a36b57e3016dd25e0fba90796c69fa.gif)](https://gyazo.com/55a36b57e3016dd25e0fba90796c69fa)

Also added a link to the home page for the logo and removed the menu items to the pages that will not be available in V1.0.

Now the PR also closes #345, #365 .
